website/docs/user_guide/token.mdx

```diff
@@ -5,7 +5,7 @@ title: How to create API Tokens
 
 import newTokenImage from '/img/admin_create_token_form.png';
 
-::: info Permissions
+:::info Permissions
```

Comment:
Not really relevant to this PR but it was broken and seemed overkill to make a new branch. Seems to be an autoformatter gone wild. Currently not working on the deployed version either: https://docs.getunleash.io/user_guide/api-token

Reply:
Great, thanks! 😄